sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ### Bugs Fixed
 
+- [Bug #20778](https://github.com/Azure/azure-sdk-for-js/pull/20778) Retry logic(throttling/exponential) didn't honor `abortSignal` that was being passed as the constructor client options of the SDKs, which would lead to longer delays and could be misinterpreted as hanging code.
```

Comment:
```suggestion
- [Bug #20778](https://github.com/Azure/azure-sdk-for-js/pull/20778) Retry logic(throttling/exponential) didn't honor `abortSignal` that was being passed as part of the constructor client options of the SDKs, which would lead to longer delays and could be misinterpreted as hanging code.
```